CONTRIBUTING.md

```diff
@@ -148,8 +148,7 @@ fetch_latest_backup('data/')
 
 
 ##### Suppliers information (CNPJ)
-1. `src/fetch_cnpj_info.py` iterates over the CEAP datasets looking for supplier unique documents (CNPJ) and creates a local dataset with each supplier info.
-1. `src/clean_cnpj_info_dataset.py` clean up and translate the supplier info dataset.
```

Comment:
Specially because we now have much more scripts in this folder compared to when we started, keeping the cleaning process in the same file comes in a good time. Great refactoring.